### ISSUE TITLE
Don't depend on react-dnd

### ISSUE THREAD

**Issue by cormacrelf** [zyzo/react-dnd-mouse-backend#9]
I'm the author of [angular-skyhook](https://github.com/cormacrelf/angular-skyhook), which is a re-implementation of `react-dnd` in Angular. It uses the same underlying `dnd-core` and is therefore compatible with all the backends.

The only problem is that existing backends tend to depend on `react-dnd`, and by extension, `react`. Technically, backends don't need any dependencies, but a `peerDependency` on `dnd-core` would be appropriate if you wanted to, and useful if it ever adds type annotations. Browsing this codebase, it seems like the only use of `react-dnd` is for the examples, which means the only work to do is change how the examples refer to the backend. You could probably just find and replace to make all references start with `../../` or something.

I'm happy to submit a PR to that effect.

Related issues

- https://github.com/LouisBrunner/dnd-multi-backend/issues/21
- https://github.com/yahoo/react-dnd-touch-backend/issues/96

**Comment by zyzo** [comment]
@cormacrelf you're right. And thank you for the PR, however I prefer to put is as peerDependencies just for informative purpose (react-dnd seems to went far since)

**Comment by cormacrelf** [comment]
Mate! Close but no cigar. It would be fine to peerDepend on `dnd-core`, but requiring `react-dnd` just brings us back to square one for `angular-skyhook`.

**Comment by zyzo** [comment]
Ah ok, didn't get you the first time. I'll fix that in next release

**Comment by cormacrelf** [comment]
Brilliant, thank you!
